calcom/cal.com
# fix: event description for teams on mobile

## What does this PR do?

Unclear if I need to raise an issue before submitting a PR to fix this. The issue is that long event descriptions overflow on mobile screen sizes.  Below is a before and after. (Apologies for the massive image size)

It removes the `truncate` class on the description to remove the following style preventing the correct display
```CSS
.truncate, .whitespace-nowrap {
    white-space: nowrap;
}
```

This is replaced with `overflow-ellipsis overflow-hidden` to achieve the same affect without a hard wrap where it can grow vertically. 
This changes the layout behaviour in the public pages to have descriptions wrap onto the netflix and truncate at 100 characters, 
in the Event Type List view, the old behaviour is preserved.

It includes an ellipsis added to the end of the description if it's longer than 100, this will be hidden and replaced by the overflow ellipsis on larger screens, but removing truncate on smaller screens removes any overflow ellipsis so this adds it back.
```JS
{eventType.description.length > 100 && "..."}
```

![2021-12-06-174216_618x1185_scrot](https://user-images.githubusercontent.com/10099540/144897221-ca10c218-1f0c-4d28-99fb-399c1e28aa23.png)
![2021-12-06-174359_623x1183_scrot](https://user-images.githubusercontent.com/10099540/144897236-d59c4896-32bf-423f-9c33-f5bd3ac04bf3.png)



## Type of change




## How should this be tested?

On mobile screens ( I am using iPhone 5/SE to check), a team's event description will overflow if long enough and push the team member avatars off screen , despite the description being truncated. 

After this change the text should wrap onto multiple lines and still display ellipsis if truncated but not overflowing.

You should be able to see the difference noted in the above screenshots.




## Checklist:
